crates/router/src/connector/bankofamerica/transformers.rs

```diff
@@ -798,12 +755,16 @@ impl<F>
             }
             BankOfAmericaPaymentsResponse::ErrorInformation(error_response) => Ok(Self {
                 response: Err(types::ErrorResponse {
-                    code: consts::NO_ERROR_CODE.to_string(),
+                    code: error_response
```

Comment:
Is this value dynamic or static ?

Reply:
This value is dynamic

Reply:
Error message should be unique static enums so please map it to error reason according to [cybersource doc](https://support.cybersource.com/knowledgebase/knowledgearticle/?code=KA-04103)